### PR TITLE
Improve settings panel accessibility

### PR DESCRIPTION
**Pull request by terotik**

### CODE REVIEW

File: src/components/general/GoalOutcomeBar.tsx [hunk 1]
```diff
@@ -170,7 +170,40 @@ const BarWithLabel = (props) => {
   );
 };
 
-const GoalOutcomeBar: React.FC<{}> = (props) => {
+const OutcomeAsText = (
```
Comment (woodwoerk) [inline src/components/general/GoalOutcomeBar.tsx:173]: Should this be camel cased?

File: src/components/general/MediumSettings.tsx [hunk 2]
```diff
@@ -57,16 +57,16 @@ const StyledOutcomeCol = styled(Col)`
...
+const YearRangeSelector = (props) => {
+  const { minYear, maxYear, referenceYear } = props;
+  const inputReference = useRef(null);
   const [popoverOpen, setPopoverOpen] = useState(false);
   const toggle = () => setPopoverOpen(!popoverOpen);
 
+  // Focus on this when visible
+  useEffect(() => {
+    inputReference?.current?.focus();
+  }, []);
```
Comment (woodwoerk) [inline src/components/general/MediumSettings.tsx:69]: Is this working as intended? When testing locally nothing was automatically focused

File: public/locales/en/common.json [hunk 3]
```diff
@@ -88,5 +89,7 @@
   "sort-descending": "Sort Descending",
   "skip-to-main-content": "Skip to main content",
   "outcome-sub-nodes": "Subsectors",
-  "outcome-sub-nodes-secondary": "Subsectors"
+  "outcome-sub-nodes-secondary": "Subsectors",
+  "outcome-bar-summary-historical": "{{goalType}} in year {{selectedYear}} was {{selectedYearDifference}} different than the goal in year {{nearestGoalYear}} ({{nearestGoalValue}}).",
+  "outcome-bar-summary-forecast": "With the chosen scenario of {{scenarioName}} the forecast of {{goalType}} in {{selectedYear}} is {{selectedYearValue}}. Target for the year {{nearestGoalYear}} is {{nearestGoalValue}}."
```
Comment (woodwoerk) [inline public/locales/en/common.json:94]: Nitpick: There's some double punctuation going on here, we could strip trailing fullstops before passing them to this translation. No need to do this now though.

Screenshot 2023-10-18 at 9 55 46

**Review by woodwoerk — APPROVED** [review]
Nice work with those outcome translations

File: public/locales/en/common.json (hunk 3 above)
Comment (terotik) [reply public/locales/en/common.json:94]: Yeah noticed that too. I thought as well to leave the trimming to later times.

File: src/components/general/MediumSettings.tsx (hunk 2 above)
Comment (terotik) [reply src/components/general/MediumSettings.tsx:69]: Before you could not access the modal with keyboard at all. Now the focus goes to modal card itself so it's not visible as the card has no visual focus state.

Now you need to tab once more to focus on things inside the modal. Maybe we should add the ref to the maxYear button instead as that's the most likely thing to interact with. But then need to pass the ref a bit further...

File: src/components/general/GoalOutcomeBar.tsx (hunk 1 above)
Comment (terotik) [reply src/components/general/GoalOutcomeBar.tsx:173]: Yes!

File: src/components/general/MediumSettings.tsx (hunk 2 above)
Comment (terotik) [reply src/components/general/MediumSettings.tsx:69]: Ok this is not working anymore. I'm sure it did last night. Thanks for testing!! I'll debug

**Pull request merged**